packages/eds-lab-react/src/components/DatePicker/SelectHeader.tsx

```diff
@@ -0,0 +1,136 @@
+import styled from 'styled-components'
+import { datePicker as tokens } from './DatePicker.tokens'
+import { arrow_back, arrow_forward } from '@equinor/eds-icons'
+import { Icon, Button, NativeSelect } from '@equinor/eds-core-react'
+import { ReactDatePickerCustomHeaderProps } from 'react-datepicker'
+
+const SelectHeader: React.FC<ReactDatePickerCustomHeaderProps> = ({
```

Comment:
I am thinking maybe this could have a bit more descriptive name since it is something that can be imported tied to the datePicker. "DatePickerSelectHeader" maybe?

Reply:
if it's wanted as part of this component yes.

but keep in mind I just added this as a test example, as f.ex. "September" becomes too big of a word to fit in the select, and decreasing the font-size and margins to make it fit will probably make it non-WCAG-friendly :)
A more detailed look at it would be required to see how we can fit in all the elements, rather than me just hacking away at it. I'm not using this in our project as I've created my own custom version with specific features for us

the changes in DatePicker.tsx & Token is the important ones that enables a more flexible usage of the component